packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition_multiplier.js

```diff
@@ -0,0 +1,224 @@
+import { localize } from '@deriv/translations';
+import { getCurrencyDisplayCode, getDecimalPlaces } from '@deriv/shared';
+import DBotStore from '../../../dbot-store';
+import { runGroupedEvents, runIrreversibleEvents } from '../../../utils';
+import { config } from '../../../../constants/config';
+import ApiHelpers from '../../../../services/api/api-helpers';
+
+Blockly.Blocks.trade_definition_multiplier = {
+    init() {
+        this.jsonInit(this.definition());
+    },
+    definition() {
+        return {
+            message0: localize('Multiplier: {{ multiplier }}', {
+                multiplier: '%1',
+            }),
+            message1: `${localize('Stake')}: %1 %2`,
+            message2: '%1',
+            message3: '%1',
+            args0: [
+                {
+                    type: 'field_dropdown',
+                    name: 'MULTIPLIERTYPE_LIST',
+                    options: [['', '']],
+                },
+            ],
+            args1: [
+                {
+                    type: 'field_label',
+                    name: 'CURRENCY_LIST',
+                    text: getCurrencyDisplayCode(config.lists.CURRENCY[0]),
+                },
+                {
+                    type: 'input_value',
+                    name: 'AMOUNT',
+                    check: 'Number',
+                },
+            ],
+            args2: [
+                {
+                    type: 'input_dummy',
+                },
+            ],
+            args3: [
+                {
+                    type: 'input_statement',
+                    name: 'MULTIPLIER_PARAMS',
+                },
+            ],
+            colour: Blockly.Colours.Special1.colour,
+            colourSecondary: Blockly.Colours.Special1.colourSecondary,
+            colourTertiary: Blockly.Colours.Special1.colourTertiary,
+            previousStatement: null,
+            nextStatement: null,
+            tooltip: localize('Define your trade options such as mulitpliers and stake.'),
+            category: Blockly.Categories.Trade_Definition,
+        };
+    },
+    meta() {
+        return {
+            display_name: localize('Multiplier trade options'),
+            description: localize('Define your trade options such as mulitplier and stake.'),
```

Comment:
`tooltip` vs `description`: `multiplier` or `multipliers`?

Reply:
Also typo.

Reply:
Woops. Thank you for noticing that 💯